next-tavla/src/Admin/components/DeleteModal/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { PrimaryButton, SecondaryButton } from '@entur/button'
+import { Modal } from '@entur/modal'
+import { Heading1, LeadParagraph } from '@entur/typography'
+import { TBoard } from 'types/settings'
+import classes from './styles.module.css'
+
+function DeleteModal({
+    board,
+    isOpen,
+    closeModal,
+    onDelete,
+}: {
+    board: TBoard
+    isOpen: boolean
+    closeModal: () => void
+    onDelete: () => void
```

Comment:
Synes vi heller kan bruke `deleteHandler`, siden `onDelete` ligner veldig på navnekonvensjonen for DOM-event handlers